README.rst

```diff
@@ -64,6 +64,15 @@ Install dependencies and the ``sysl`` package with symlinks ::
 
   > pip install pytest flake8 -e .
 
+Set up your environment for diagram generation by providing `PlantUML <http://plantuml.com/>` 
```

Comment:
it may also be good to introduce PlantUML.

"Sysl depends upon PlantUML for diagram generation. Some of the automated tests require a PlantUML dependency." ....